packages/angular/src/builders/webpack-server/webpack-server.impl.ts

```diff
@@ -0,0 +1,58 @@
+import { BuilderContext, createBuilder } from '@angular-devkit/architect';
+import { JsonObject } from '@angular-devkit/core';
+import type { Schema } from './schema';
+
+import { parseTargetString, joinPathFragments } from '@nrwl/devkit';
+import { Workspaces } from '@nrwl/tao/src/shared/workspace';
+import {
+  DevServerBuilderOptions,
+  serveWebpackBrowser,
+} from '@angular-devkit/build-angular/src/dev-server';
+import { existsSync } from 'fs';
+import { merge } from 'webpack-merge';
+import { normalizeOptions } from './lib';
+
+export function webpackServer(schema: Schema, context: BuilderContext) {
+  const options = normalizeOptions(schema);
+  const workspaceConfig = new Workspaces(
+    context.workspaceRoot
+  ).readWorkspaceConfiguration();
+
+  const parsedBrowserTarget = parseTargetString(options.browserTarget);
```

Comment:
@vsavkin 
I've set this up so that it tries to fetch the custom webpack config path from the build target. That way, the dev only needs to change the builder/executor to use ours instead of angular's without changing any of the additional `serve` target's options.


Of course, in our generator we can do this for them automatically.

I didn't want to make them resupply the path to the webpack config again in this target as it felt like overkill.